src/components/ui/Loading.tsx

```diff
@@ -0,0 +1,18 @@
+import { SVGProps } from 'react';
+
+export const Loading: React.FC<SVGProps<SVGSVGElement>> = (props) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="44"
+    height="44"
+    viewBox="0 0 45 44"
+    className="animate-spin"
+    {...props}
+  >
+    <circle width="44" height="44" fill="none" />
+    <path
+      d="m31.723 14.031c1.7868 1.7499 2.7313 4.3377 2.7313 7.4836 0 3.6305-0.4945 6.7642-0.8876 8.5115h-4.2539c1.475-2.6381 1.7051-5.6827 1.7051-7.3187 0-2.2855-0.4543-4.081-1.3502-5.3365-0.8643-1.2115-2.1234-1.8786-3.5455-1.8786-1.453 0-2.6322 0.6263-3.41 1.8113-0.7086 1.0795-1.053 2.5598-1.053 4.5257 0 1.7547-0.3351 2.6334-0.6163 3.0616-0.2879 0.4389-0.6541 0.6262-1.2245 0.6262-0.5804 0-1.0443-0.2546-1.4181-0.7784-0.3902-0.5468-0.8551-1.6475-0.8551-3.7875 0-1.4469 0.2047-4.1985 1.576-6.3704 1.2706-2.0126 3.2778-3.0332 5.9657-3.0332 2.6825 0 4.9772 0.8588 6.6361 2.4834zm-17.505 15.597c-1.7869-1.7499-2.7312-4.3377-2.7312-7.4836 0-3.6305 0.4944-6.764 0.8875-8.5115h4.2538c-1.4749 2.6381-1.705 5.6826-1.705 7.3187 0 2.2855 0.4543 4.081 1.3501 5.3365 0.8644 1.2115 2.1236 1.8786 3.5455 1.8786 1.4532 0 2.6323-0.6263 3.4101-1.8113 0.7086-1.0795 1.053-2.5598 1.053-4.5257 0-1.7547 0.3351-2.6334 0.6162-3.0617 0.2881-0.4388 0.6543-0.6261 1.2245-0.6261 0.5806 0 1.0444 0.2546 1.4183 0.7784 0.39 0.5466 0.8551 1.6475 0.8551 3.7875 0 1.4469-0.2048 4.1985-1.576 6.3704-1.2708 2.0127-3.2778 3.0332-5.9657 3.0332-2.6825 0-4.9772-0.8588-6.6362-2.4834zm4.5869-18.644h-8.4254l-0.2789 0.9465c-0.0505 0.1712-1.2363 4.2616-1.2363 10.214 0 3.8796 1.2201 7.1251 3.5284 9.3857 2.1272 2.0831 5.1322 3.2304 8.4615 3.2304 2.4979 0 4.6258-0.7133 6.2821-2.0848h8.4254l0.279-0.9465c0.0505-0.1712 1.2362-4.2616 1.2362-10.214 0-3.8794-1.2201-7.1251-3.5284-9.3857-2.1271-2.0831-5.132-3.2304-8.4613-3.2304-2.4979 0-4.626 0.71338-6.2823 2.0848z"
+      fill="#00E526"
+    />
+  </svg>
+);
```

Comment:
I am not sure we need every single svg as a react component. If yes. Let's use react-svgr

Reply:
ok!